### PR TITLE
Implement a basic tooltip

### PR DESCRIPTION
**Pull request by mastertheblaster**

### CODE REVIEW

**Comment by mastertheblaster** [comment]
Missing a bunch of tests for the tooltip, will add later.

File: src/Tooltip/Tooltip.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      onClick: this._chainCallbacks(child.onClick, this._onClick),
+      onMouseEnter: this._chainCallbacks(child.onMouseEnter, this._onMouseEnter),
+      onMouseLeave: this._chainCallbacks(child.onMouseLeave, this._onMouseLeave),
+      onFocus: this._chainCallbacks(child.onFocus, this._onFocus),
+      onBlur: this._chainCallbacks(child.onBlur, this._onBlur)
+    });
+  }
+
+  _chainCallbacks = (first, second) => {
+    return args => {
```
Comment (argshook) [inline src/Tooltip/Tooltip.js]: @mastertheblaster i guess here could be multiple, so maybe `(...args)`?

**Comment by 3LOK** [comment]
Hi, I've merged - but note that the current 'bounce' method is unpractical for single bounces people might want.

**Comment by mastertheblaster** [comment]
@3LOK I will introduce a method `bounceOnce` which will do all the logic.
And tooltip will work in controlled and uncontrolled manner.

**Pull request merged**